packages/toc-extension/src/index.ts

```diff
@@ -93,9 +94,11 @@ async function activateTOC(
     label: trans.__('Run Cell(s)')
   });
 
-  app.contextMenu.addItem({
-    selector: '.jp-tocItem',
-    command: CommandIDs.runCells
+  app.commands.addCommand(CommandIDs.showPanel, {
```

Comment:
Looks like this command was introduced in https://github.com/jupyterlab/jupyterlab/pull/10425, but not yet backported to 3.2.x.

Reply:
So probably we could backport it separately when backporting #10425 for consistency.